website/client/src/components/admin-panel/user-support/index.vue

```diff
@@ -0,0 +1,175 @@
+        />
+
+        <privileges-and-gems
+          :hero="hero"
+          :reset-counter="resetCounter"
+        />
+
+        <cron-and-auth
+          :hero="hero"
+          :reset-counter="resetCounter"
+        />
+
+        <party-and-quest
+          :user-id="hero._id"
+          :username="hero.auth.local.username"
+          :user-has-party="hasParty"
+          :party-not-exist-error="partyNotExistError"
+          :user-party-data="hero.party"
+          :group-party-data="party"
+          :reset-counter="resetCounter"
+        />
+
+        <avatar-and-drops
+          :items="hero.items"
+          :preferences="hero.preferences"
+        />
+
+        <items-owned
+          :hero="hero"
+          :reset-counter="resetCounter"
+        />
+
+        <contributor-details
+          :hero="hero"
+          :reset-counter="resetCounter"
+          @clear-data="clearData"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss">
+  .accordion-group .accordion-group {
+    margin-left: 1em;
+  }
+  h3 {
+    margin-top: 2em;
+  }
+  h4 {
+    margin-top: 1em;
+  }
+  .expand-toggle::after {
+    margin-left: 5px;
+  }
+  .subsection-start {
+    margin-top: 1em;
+  }
+  .form-inline {
+    margin-bottom: 1em;
+    input, span {
+      margin-left: 5px;
+    }
+  }
+  .errorMessage {
+    font-weight: bold;
+  }
+  .markdownPreview {
+    margin-left: 3em;
+    margin-top: 1em;
+  }
+</style>
+
+<script>
+import BasicDetails from './basicDetails';
+import ItemsOwned from './itemsOwned';
+import CronAndAuth from './cronAndAuth';
+import PartyAndQuest from './partyAndQuest';
+import AvatarAndDrops from './avatarAndDrops';
+import PrivilegesAndGems from './privilegesAndGems';
+import ContributorDetails from './contributorDetails';
+
+import { mapState } from '@/libs/store';
+
+export default {
+  components: {
+    BasicDetails,
+    ItemsOwned,
+    CronAndAuth,
+    PartyAndQuest,
+    AvatarAndDrops,
+    PrivilegesAndGems,
+    ContributorDetails,
+  },
+  data () {
+    return {
+      userIdentifier: '',
+      resetCounter: 0,
+      hero: {},
+      party: {},
+      hasParty: false,
+      partyNotExistError: false,
+    };
+  },
+  computed: {
+    ...mapState({ user: 'user.data' }),
+  },
+
+  watch: {
+    userIdentifier () {
+      // close modal if the page is opened in an existing tab from the modal
+      this.$root.$emit('habitica::dismiss-modal', 'profile');
+
+      this.loadHero(this.userIdentifier);
+    },
+  },
+  mounted () {
+    this.userIdentifier = this.$route.params.userIdentifier;
+  },
+  methods: {
+    clearData () {
+      this.hero = {};
+    },
+
+    async loadHero (userIdentifier) {
+      const id = userIdentifier.replace(/@/, ''); // allow "@name" to be entered
+      this.$emit('changeUserIdentifier', id); // change user identifier in Admin Panel's form
+
+      const hero = await this.$store.dispatch('hall:getHero', { uuid: id });
+      this.hero = { ...hero };
```

Comment:
otherwise it'll only update here because props flow downstream, see https://vuejs.org/v2/guide/components-props.html#One-Way-Data-Flow

Reply:
I've made the other changes locally but for this one, I wouldn't mind leaving it as it is if that's okay with you? Once other features are added to the admin panel, such as the CMS, it will be neater if all the code for user support is in one sub-directory. 
(I think it still makes sense for the search form to be on the top level page so that you can access it as soon as you reach the admin panel.)

Reply:
Sorry, I got confused, I thought `hero` was passed as a prop to this component while it's here that it's fetched the first time and then passed to the other components, so that's fine